### PR TITLE
Implement component NavBarItem

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement component NavBarItem

image

image

Storybook : https://6684faab2d0c99a3e93bf145-xiloatdzqw.chromatic.com/

### CODE REVIEW

File: src/components/NavBarItem/navBarItem.module.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.listItem {
```
Comment (thanhnguyen-agilityio) [inline src/components/NavBarItem/navBarItem.module.css]: css naming convention, pls recheck.

File: src/components/NavBarItem/navBarItem.module.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  list-style-type: none;
+  font-family: Mada;
+}
+
+.link {
```
Comment (thanhnguyen-agilityio) [inline src/components/NavBarItem/navBarItem.module.css]: Is this general, put into generic class to reuse.

File: src/components/NavBarItem/navBarItem.module.css (hunk 1 above)
Comment (thanhtrantruong47) [reply src/components/NavBarItem/navBarItem.module.css]: Fixed

File: src/components/NavBarItem/navBarItem.module.css (hunk 2 above)
Comment (thanhtrantruong47) [reply src/components/NavBarItem/navBarItem.module.css]: Fixed

File: src/styles/genericStyles.module.css
Comment (thanhnguyen-agilityio) [inline src/styles/genericStyles.module.css:1]: Add folder for module css:
`styles/modules/utilities.module.css`

File: src/components/NavBarItem/NavBarItem.tsx [hunk 3]
```diff
@@ -1,11 +1,22 @@
+import { Link } from 'react-router-dom';
+import utils from '../../styles/utils.module.css';
```
Comment (thanhnguyen-agilityio) [inline src/components/NavBarItem/NavBarItem.tsx]: utils -> utilStyles

Comment (thanhtrantruong47) [reply src/components/NavBarItem/NavBarItem.tsx]: Fixed

File: src/styles/genericStyles.module.css
Comment (thanhtrantruong47) [reply src/styles/genericStyles.module.css:1]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**